winpr/include/winpr/wtsapi.h

```diff
@@ -1323,10 +1323,10 @@ typedef HANDLE(WINAPI* WTS_VIRTUAL_CHANNEL_OPEN_EX_FN)(DWORD SessionId, LPSTR pV
 typedef BOOL(WINAPI* WTS_VIRTUAL_CHANNEL_CLOSE_FN)(HANDLE hChannelHandle);
 
 typedef BOOL(WINAPI* WTS_VIRTUAL_CHANNEL_READ_FN)(HANDLE hChannelHandle, ULONG TimeOut,
-                                                  PCHAR Buffer, ULONG BufferSize,
+                                                  LPVOID lpBuffer, ULONG BufferSize,
                                                   PULONG pBytesRead);
```

Comment:
Just curious why are you changing the types here, as it will have some consequences on the other implementation of WTS (namely ogon)

Reply:
It's not so important, just a compiler warning. `WTS_VIRTUAL_CHANNEL_READ_FN` is set to `Win32_WTSVirtualChannelRead` which calls `ReadFile()` which has void* as 2nd parameter. GCC shows warning about function pointer mismatch. If this change creates any major issue, I can revert it.

Reply:
Ok so I guess the change should be done in ` Win32_WTSVirtualChannelRead` then

Reply:
OK, I will remove the change.